brightchain-lib/src/lib/encryptedBlockMetadata.ts

```
@@ -1,58 +1,118 @@
-  ): EncryptedBlockMetadata {
-    return new EncryptedBlockMetadata(
-      metadata.size,
-      metadata.type,
-      metadata.dataType,
-      metadata.lengthWithoutPadding,
-      metadata.creator,
-      metadata.dateCreated,
-    );
+  public static override fromJsonValidator(data: any): void {
+    // Call parent validator first
+    super.fromJsonValidator(data);
```

Biome (1.9.4)

[error] Using super in a static context can be confusing.

super refers to a parent class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)